### DISCUSSION TITLE
Error message when attempting to create Search Feed

### DISCUSSION THREAD

**Discussion by podworx** [rrirower/Channel-Feed-Manager#113] [Q&A]
When trying to create a search feed for a channel, each entry generates a FATAL message:

Program exception detected: 'The given key 'searchLicense' was not present in the dictounary.' in method 'AppendSearchFeed'.

Any thoughts on this, folks?

**Comment by rrirower** [comment]
What version are you using?

**Reply by podworx** [reply]
ChannelFeedManager Version 1.0.0.7

**Reply by rrirower** [reply]
Did you uninstall the previous version before installing 1.0.0.7?  I thought this bug had been fixed, but, it may have new symptoms.

**Reply by podworx** [reply]
Yes, sir.  I did.

**Reply by rrirower** [reply]
I'll need to investigate further.

**Reply by rrirower** [reply]
I created search feeds for several feed files including a previous feed you provided.  I received some warnings on your feed because some values were not defined in the feed. I did not receive the error you posted.

Can you post the actual feed file you are using?

**Reply by podworx** [reply]
Yep.  

http://pdwxroku.com/cfs/roku.json

**Reply by podworx** [reply]
https://scottwhitneyacademy.com/roku/feed.json

**Comment by podworx** [comment]
Also, if I add a rating pair to a video, "Update this Content", then "Save My Channel", when I open the video info up again, the rating is gone.

**Reply by rrirower** [reply]
Can you double-check this?  I used a prior feed of yours and was able to save the ratings without a problem.

**Reply by rrirower** [reply]
Also, what type of content are you editing? Movie, series, etc?

**Reply by podworx** [reply]
Movies.  Tried on two different feeds, same issue with losing the ratings.

**Reply by rrirower** [reply]
Can you possibly take a screen capture video while demonstrating your issue?

**Reply by rrirower** [reply]
Please see my screen capture:


https://github.com/rrirower/Channel-Feed-Manager/assets/34550407/fa52a7f4-174f-4a44-8688-7ce138141290

**Reply by rrirower** [reply]
Have you tried the resolution I reported in the associated issue #114 ?

**Reply by podworx** [reply]
I un-installed/reinstalled. . .same issue.  I see you screen share shows it working.  But mine still has the problem.

**Reply by rrirower** [reply]
Please try the steps in #114.
